gradoop-data-integration/src/main/java/org/gradoop/dataintegration/importer/rdbmsimporter/tuples/NameTypeTuple.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradoop.dataintegration.importer.rdbmsimporter.tuples;
+
+import org.apache.flink.api.java.tuple.Tuple2;
+
+import java.sql.JDBCType;
+
+/**
+ * Tuple representing a key string and belonging data type pair
+ */
+public class NameTypeTuple extends Tuple2<String, JDBCType> {
+
+  /**
+   * serial version uid
+   */
+  private static final long serialVersionUID = 1L;
+
+  /**
+   * Key string
+   */
+  private String name;
+
+  /**
+   * JDBC data type
+   */
+  private JDBCType type;
+
+  /**
+   * Empty Constructor
+   */
+  public NameTypeTuple() {
+  }
+
+  /**
+   * Constructor
```

Comment:
refactor all "Constructor" comments to something more viable.

Reply:
Done